asm/asm.go

```diff
@@ -1,17 +1,22 @@
-//go:build ignore
-// +build ignore
-
 package main
 
 import (
 	. "github.com/mmcloughlin/avo/build"
 	"github.com/mmcloughlin/avo/buildtags"
 	. "github.com/mmcloughlin/avo/operand"
 	. "github.com/mmcloughlin/avo/reg"
-	shared "github.com/pjbgf/sha1cd/internal"
 )
 
-//go:generate go run sha1cdblock_amd64_asm.go -out sha1cdblock_amd64.s
+const (
+	// Constants for the SHA-1 hash function.
+	RoundConst0 = 0x5A827999
+	RoundConst1 = 0x6ED9EBA1
+	RoundConst2 = 0x8F1BBCDC
+	RoundConst3 = 0xCA62C1D6
```

Comment:
Note they can't be named `K0, K1, ...` anymore since these are registers in `avo` (AVX-512 mask registers).